src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Features/ValidationOverride.feature

```diff
@@ -0,0 +1,44 @@
+﻿@database
+@api
+@domainMessageHandlers
+@messageBus
+@activeCalendarPeriod
+Feature: ValidationOverride
+	In order to prevent incorrect payment validation results
+	As a support user
+	I want to be able to override payment validations
+
+Scenario Outline: A payment validation override is created
+	Given an apprenticeship incentive has a <validationType> validation
+	When the validation override request is received
+	Then the validation override <validationType> is stored against the apprenticeship incentive
+
+Examples:
+    | validationType                  |
+    | HasBankDetails                  |
```

Comment:
According to the JIRA ticket, we shouldn't support all validation types. Overriding HasBankDetails would cause a failure in the payment process because the employer couldn't be paid

Reply:
The allowable overrides are restricted in the Functions so HasBankDetails etc.. should never get called.  I didn't duplicated that here so that could be easily changed.

Reply:
Updated